Firestore/core/test/firebase/firestore/model/CMakeLists.txt

```diff
@@ -21,7 +21,6 @@ cc_test(
     field_mask_test.cc
     field_path_test.cc
     field_value_test.cc
-    maybe_document_test.cc
```

Comment:
Why remove this unit test?

Reply:
Because MaybeDocument is now abstract, so you can't test it directly. Fortunately (?) all the tests were already duplicated in the subclass tests.